signature-specification.md

```diff
@@ -248,6 +249,47 @@ The signing time denotes the time at which the signature was generated. A X509 c
 
 This is an optional feature that provides a “best by use” time for the artifact, as defined by the signer. Notary v2 allows users to include an optional expiry time when they generate a signature. The expiry time is not set by default and requires explicit configuration by users at the time of signature generation. The artifact is considered expired when the current time is greater than or equal to expiry time, users performing verification can either configure their trust policies to fail the verification or even accept the artifact with expiry date in the past using policy. This is an advanced feature that allows implementing controls for user defined semantics like deprecation for older artifacts, or block older artifacts in a production environment. Users should only include an expiry time in the signed artifact after considering the behavior they expect for consumers of the artifact after it expires. Users can choose to consume an artifact even after the expiry time based on their specific needs.
 
+### Signature Portability
+
+Portability of signatures is associated with the portability of associated artifacts which are being signed.
+OCI artifacts are inherently location agnostic, artifacts can be pulled from and pushed to any OCI compliant registry to which a user has access. 
+The artifacts themselves can be classified as follow.
+
+1. *Public Artifacts* -  Artifacts that are distributed publicly for broad consumption. 
+Artifacts distributed via public registries fall in this category. 
+E.g. Public images for software distributed by software vendors, and open source projects. 
+Signatures associated with these artifacts require broad portability.
+1. *Private Artifacts* - Artifacts that are private to a user or organization, and may be shared with limited parties. 
```

Comment:
What is the intent when users/customers are multi-cloud/hybrid with on-prem?

Reply:
I've simplified the language used in this section without referring all reasons an artifact may not be portable.